scripts/upgrade.sh.in

```diff
@@ -152,7 +181,10 @@ run "Performing Pre-Upgrade checks" precheck_upgrade
 run "Upgrading bootstrap" upgrade_bootstrap
 run "Setting cluster version to $DESTINATION_VERSION" patch_kubesystem_namespace
 run "Launching the pre-upgrade" launch_pre_upgrade
-run "Launching the upgrade" launch_upgrade
+run "Upgrading ETCD cluster" upgrade_etcd
+run "Upgrading all APIServers" upgrade_apiservers
+run "Upgrading bootstrap engines" upgrade_local_engines
```

Comment:
What are `bootstrap engines`? `Upgrading local containerd and kubelet` or something alike?

Reply:
Oh yes I wanted to put "Upgrading bootstrap container engines" but yes I can put "local" one here